BoneReconstructionPlanner/BoneReconstructionPlanner.py

```diff
@@ -733,19 +816,18 @@ def process(self,fibulaLine,mandibularCurve,planeList,initialSpace,betweenSpace)
         dynamicModelerNode.AddNodeReferenceID("PlaneCut.InputPlane", planeList[0].GetID()) 
       dynamicModelerNode.SetNodeReferenceID("PlaneCut.OutputPositiveModel", modelNode.GetID())
       dynamicModelerNode.SetAttribute("OperationType", "Difference")
-      self.planeCutsList.append(dynamicModelerNode)
 
-      shNode.CreateItem(self.planeCutsFolder,dynamicModelerNode)
-      shNode.CreateItem(self.cutBonesFolder,modelNode)
+      shNode.CreateItem(planeCutsFolder,dynamicModelerNode)

```

Comment:
Why do you need to manually create items? It should not be necessary.

Reply:
I use the function CreateItem to insert an item I created inside a folder I choose. I don't know if a better way to do this

Reply:
To move a SH item into a folder, set its parent item by calling SetItemParent.